### ISSUE TITLE
Reduce the file size

### ISSUE THREAD

**Issue by takahirox** [MozillaReality/WebXR-emulator-extension#152]
Currently the total extension file size is about 580KB when compressed and 2.2MB when uncompressed. It'd be nice if we can reduce the file size.

The majority of it is
* Three.js
* webxr-polyfill
* Assets

Potential solutions may be
* Three.js tree shaking
* Remove VRDisplay and WebVRDevice from webxr-polyfill #143
* Use lighter or well compressed (eg. with draco compression) asset models

**Comment by feiss** [comment]
just made a test and `oculus-go-controller` can be reduced to 234kb by exporting to `glb`, and removing unnecessary stuff like uvs, materials and texture (which don't add too much to the appearance).

I'll try to send a PR tomorrow with this optimization

**Comment by feiss** [comment]
btw, it would be amazing if three had a build system so you could specify which modules you want. Something like old jquery-ui (https://jqueryui.com/download/)

**Comment by fernandojsg** [comment]
I believe it already does tree shaking if you use modules

**Comment by feiss** [comment]
ok, I couldn't help it and I made a low resolution version of the oculus-go-controller:

<img width="705" alt="Screenshot 2019-11-16 at 01 16 28" src="https://user-images.githubusercontent.com/359872/68984128-cc65a800-080e-11ea-8325-40396b8f40bc.png">

It's lower quality and resolution, but I think it's more than enough for our purposes. And it's 13KB instead of the original 540KB.

It's pretty late now here and I don't have time for a PR. I attached here the GLB file. You just have to load it and apply a MeshLambertMaterial on all the objects inside. I've also included the .blend file.

[oculus-go-controller.zip](https://github.com/MozillaReality/WebXR-emulator-extension/files/3853612/oculus-go-controller.zip)
